src/utils/tests/dispatcher.test.js

```diff
@@ -5,9 +5,15 @@ describe('parseMessage', () => {
   test('client emits command event when command is valid', () => {
     const emitterSpy = jest.spyOn(client, 'emit');
     parseMessage('++faq');
-    expect(emitterSpy).toHaveBeenCalledWith('command', 'faq', []);
```

Comment:
A bot is generally a client receive/respond with an input string and outputting something as a result right?  So it doesn't seem like you'd need to actually spy on the client.  Unless it's responding through a rest / http call.  Then I'd separate that into another function that's totally independent of everything and make all the other functionality testable.

Reply:
It doesn't output anything as a result in this case. It's emitting and doesn't return anything.